### DISCUSSION TITLE
New tuya airco support

### DISCUSSION THREAD

**Discussion by basdelouw78** [make-all/tuya-local#2021] [Q&A]
I've rewrite a new tuya support for the qlima airco. How can i add this to the new tuya update?

**Comment by make-all** [comment]
Seems offtopic, or you haven't expressed your request clearly enough.

**Reply by basdelouw78** [reply]
Ok, sorry not clear enough i think. I've write a yaml to support my qlima airco. How can i upload the code so it can be in the new release?
